packages/webpack-build-scripts/webpack.config.base.mjs

```diff
@@ -179,7 +177,7 @@ export default {
                             legacyDecorator: true,
                             react: {
                                 refresh: !IS_PRODUCTION,
-                                runtime: "classic",
+                                runtime: "automatic",
```

Comment:
Setting this to "automatic" uses the new JSX runtime module

https://swc.rs/docs/configuration/compilation#jsctransformreactruntime

> Use `runtime: automatic` to use a JSX runtime module (e.g. `react/jsx-runtime` introduced in React 17).